libs/dal/src/repositories/message/message.schema.ts

```diff
@@ -54,6 +54,7 @@ const messageSchema = new Schema<MessageDBModel>(
             content: Schema.Types.String,
             resultContent: Schema.Types.String,
             url: Schema.Types.String,
+            target: Schema.Types.String,
```

Comment:
missing the schema type, without this piece the `target` field is not saved in Mongo :/